packages/itwin/property-grid/src/components/MultiElementPropertyGrid.tsx

```diff
@@ -112,41 +105,32 @@ export const MultiElementPropertyGrid = (props: PropertyGridProps) => {
       disableUnifiedSelection={true}
       onBackButton={() => {
         setContent(MultiElementPropertyContent.ElementList);
-        setAnimationForward(false);
       }}
       key={"SingleElementPropertyGrid"}
     />
   );
 
+  // since css animation requires all react components be rendered - just move non-active components to the side
+  const getAnimationStyle = (idx: MultiElementPropertyContent.PropertyGrid) => {
```

Comment:
i dont think the type for this should be of `MultiElementPropertyContent.PropertyGrid`. can probably just do number